### ISSUE TITLE
maegatk-indel: command not found...

### ISSUE THREAD

**Issue by zjy0419** [caleblareau/maegatk#6]
Hi,

I was wondering the relationship between maegatk and maegatk-indel, is the output of the former the input of the latter? 

In addition, after installing this package by 'pip3 install maegatk' ,I can use maegatk, but cannot use maegatk-indel (bash: maegatk-indel: command not found...). What should I do to make it work?

Thank you very much!

**Comment by caleblareau** [comment]
@vincent6liu would you mind commenting here?

**Comment by pl-ki** [comment]
Same installation issue here - 'maegatk-indel: command not found'.

**Comment by caleblareau** [comment]
I was able to clone the repo and install maegatk-indel via `pip install -e .` for a new version of python: 

```
python --version
Python 3.11.6

maegatk-indel --help
Usage: maegatk-indel [OPTIONS]

  maegatk: a Maester genome toolkit.

  INDEL calling
```

I'm not sure the best way to proceed here at the moment...

**Comment by amberqiong** [comment]
I have the same issue. I can run maegatk successfully at python 3.6.8, with snakemake 3.13.3 but maegatk-indel is not available.  Then in the new env, maegatk-indel is available under python 3.11.10, but snakemake 8.25.3 give me the error when running maegatk that, `snakemake: error: unrecognized arguments: --stats` which snakemake should I use then to be compatible with python 3.11 and also support `--stats` argument?

**Comment by amberqiong** [comment]
> I have the same issue. I can run maegatk successfully at python 3.6.8, with snakemake 3.13.3 but maegatk-indel is not available. Then in the new env, maegatk-indel is available under python 3.11.10, but snakemake 8.25.3 give me the error when running maegatk that, `snakemake: error: unrecognized arguments: --stats` which snakemake should I use then to be compatible with python 3.11 and also support `--stats` argument?

Tested. Snakemake 7.24.2 works. 

`python --version`
`Python 3.11.10`
`snakemake --version`
`7.24.2`
